pyflakes/checker.py

```diff
@@ -1699,7 +1704,33 @@ def CALL(self, node):
                     with self._enter_annotation():
                         self.handleNode(keyword.value, node)
 
-        self.handleChildren(node)
+        if _is_typing(node.func, "TypedDict", self.scopeStack):
+            # TypedDict("a", {"a": int})
+            if len(node.args) > 1 and isinstance(node.args[1], ast.Dict):
+                self._annotated_nodes += node.args[1].values
+
+            # TypedDict("a", a=int)
+            self._annotated_nodes += [k.value for k in node.keywords]
+
+            with self._enter_annotation(AnnotationState.NONE):
+                self.handleChildren(node)
+
+        elif _is_typing(node.func, "NamedTuple", self.scopeStack):
+            # NamedTuple("a", [("a", int)])
+            if len(node.args) > 1 and isinstance(
+                node.args[1], (ast.Tuple, ast.List)
+            ) and all(isinstance(x, (ast.Tuple, ast.List)) and
+                      len(x.elts) == 2 for x in node.args[1].elts):
```

Comment:
this indentation is all over the place and is unreadable.  please write it like:

```
if (
        len(node.args) >1 and
        isinstance(node.args[1], (ast.Tuple, ast.List)) and
        all(...)
):
    # code here
```